bedrock/foundation/templates/foundation/trademarks/distribution-policy.html

```diff
@@ -92,7 +92,13 @@ <h2>{{ _('Modified Versions Require Prior Written Permission') }}</h2>
 
   <p>
     {% trans permissions="mailto:trademark-permissions@mozilla.com" %}
-    If you wish to distribute a modified version of Firefox with Mozilla trademarks please contact us with your request at <a href="{{ permissions }}">trademark-permissions@mozilla.com</a>.
+    If you wish to distribute a modified version of Firefox or other Mozilla software with Mozilla trademarks please contact us with your request at <a href="{{ permissions }}">trademark-permissions@mozilla.com</a>.
+    {% endtrans %}
+  </p>
+  <h4>{{ _('Mozilla General Trademark Guidelines') }}</h4>
+  <p>
+    {% trans click_here=url('foundation.trademarks.policy') %}
+    <a href="{{ click_here }}" >Click here</a> for the general Mozilla Trademark Guidelines.
```

Comment:
Not a blocker, but `click here` is sometimes not very useful descriptive text for a11y. Could this sentance be reworded so that the text provides some context for the target destination?

Reply:
Asked for different language